news/2020-05-08-mappers-report-april.md

```diff
@@ -0,0 +1,193 @@
+
+A map of his that truly shines in many people's eyes is his semi-recently ranked rendition of [Cyte - Command Prompt](https://osu.ppy.sh/beatmapsets/1068145#taiko/2236237). On the top difficulty (mind bogglingly named "C:\Program Files\Difficulties\434f525255505445442046494c45.exe"), Idealism proves that even when mapping genres out of his comfort zone he can still shine. 
+
+Some of his other notable and equally as great maps would be his renditions of [Blue on Blue (TamorlaM Btlg Remix)](https://osu.ppy.sh/beatmapsets/965135#taiko/2020395) and [White Eternity](https://osu.ppy.sh/beatmapsets/992443#taiko/2075446), where he shows off his addiction to eroge music through his solid mapping! We hope to see even more of his maps coming into the ranked section in the rest of 2020, as his potential to shine even brighter clearly shows!
+
+### Yuemiao
+
+Recently making her first appearance in the ranked section with her take on [Mizutama by YUC'e and Nor](https://osu.ppy.sh/beatmapsets/1009290#taiko/2112927),  **[Yuemiao](https://osu.ppy.sh/users/4493348)** shows us that she definitely knows what she's doing.
+
+Executing many different snaps and rhythms perfectly, with clean consistency throughout her difficulties, making this map definitely one of the most interesting ones in the ranked section as of late.
+Adding her own little rhythms with an intuitive yet interesting kat placement, really shows how good osu!taiko can sound and how it's possible to enhance the song that is mapped when someone knows their craft!
+
+Her ability to also map faster and more chaotic songs just goes to show that she can handle multiple genres and definitely can create something clean for everyone's taste. Try it yourself with this well polished unranked set of [IZANA - t+pazolite vs. P*Light](https://osu.ppy.sh/beatmapsets/1100797#taiko/2299870), which also includes very smooth SV sections, making this pleasant to the eye as well.
+Yuemiao definitely deserves more credit, as she is quite unknown and surprised many with her first ranked map. The current 28 ratings on her Mizatuma consist of 27(!) 10* votes. If ratings could talk, we wouldn't have to phrase this.
+We are definitely looking forward to more of her maps in the future and expect something great!
+
+The next month will feature osu!catch. But that's not all yet. We want to give every player the chance to raise their voice for the Mappers' Showcase. That's why we created this [form](https://forms.gle/kM5SW33xp1HttS7D8) where everyone can submit mappers and storyboarder of which they think they should be featured in one of the next issues of the Mappers' Report!
+
+We're excited as you to see which gems are still hidden in osu!
+
+## <a id="mapping-contests"></a>Mapping Contests
+ 
+Flex your mapping muscles by voting and participating in some of this month's ongoing contests! Contest veterans, however, might find it more appropriate to release tension as they see some of this month's mapping contest results.
```

Comment:
I'm not completely sure what that second sentence is trying to convey :P

Reply:
old contest results are published later in the section